tools/ui-components/src/button/button.stories.tsx

```diff
@@ -5,7 +5,20 @@ import { Button, ButtonProps } from '.';
 
 const story = {
   title: 'Example/Button',
-  component: Button
+  component: Button,
+  parameters: {
+    controls: {
+      include: ['children', 'variant', 'size']
+    }
```

Comment:
We extend the Button's interface from HTML attributes in order to accept attributes such as aria-disabled, aria-label, etc. This turns the Storybook controls into a clutter. 

https://github.com/freeCodeCamp/freeCodeCamp/blob/1c8659fe00d2f8b3b6bc1ce94977cccf59178c07/tools/ui-components/src/button/types.ts#L7

What I'm doing here is filtering the controls and only show what we are interested in. 

Ref: https://storybook.js.org/docs/react/essentials/controls#filtering-controls